nextjs-13/src/components/CreateNewsletterModal/index.test.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { toast } from 'react-toastify';
+
+import { render, fireEvent, screen, waitFor } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+
+import requestManager from 'lib/request/manager';
+
+import CreateNewsletterModal from './index';
+
+jest.mock('react-toastify', () => ({
+  toast: {
+    success: jest.fn(),
+    error: jest.fn(),
+  },
+  ToastContainer: jest.fn(),
+}));
+
+jest.mock('lib/request/manager', () => jest.fn());
+
+// refs: https://github.com/Andarist/react-textarea-autosize#how-to-test-it-with-jest-and-react-test-renderer-if-you-need-ref
+jest.mock('react-textarea-autosize', () => {
+  const TextareaAutosizeMock = jest.fn((props) => (
+    <textarea
+      id="content"
+      onChange={(event) => {
+        props.onChange(event); // Call the provided onChange callback
+      }}
+      value={props.value}
+    />
+  ));
+  return TextareaAutosizeMock;
```

Comment:
I think you can add line break above this `return`

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/106/commits/b5c4314bf5475947150344015ba6bcdc399813cc#diff-8a0a03315819434e30ec3761666c2fd68166d67d0332a15196e7f69df14e5eb7R32)